### ISSUE TITLE
Confused in Distributed Training in Tensorflow

### ISSUE THREAD

**Issue by 282866787** [tensorflow/tensorflow#21227]
Hi,I have tried several methods to build my distributed training system , but the training speed is fallacious. As I want, several machines with one master and muti ps(3 ps),the training speed depends on the slowest one of the ps. But what I got it is almost 200s every 100steps in En-De training, I'm very confused about this. What's more ,the t2t defines the master and ps , but in conventional tensorflow distributed training tutorial, there should be worker and ps. The ps plays the role of updating variables like master in T2T , and the worker computes the gradients parallel like ps in T2T. This really makes me amazing! I have ask questions in gitter ,but no one answer me . I hope you can give me some advice !

**Comment by 282866787** [comment]
I just want to seek the real comprehension among the tensorflow and tensor2tensor.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @yuefengz: It has been 15 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @yuefengz: It has been 46 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by yuefengz** [comment]
I think the question fits more on StackOverflow. To use multi-worker training with distribution strategies, you can take a look at our (documentations)[https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/distribute/README.md] and (examples)[https://github.com/tensorflow/ecosystem/tree/master/distribution_strategy].
